sentry-android-replay/src/main/java/io/sentry/android/replay/capture/BaseCaptureStrategy.kt

```diff
@@ -280,4 +317,15 @@ internal abstract class BaseCaptureStrategy(
             }
         }
     }
+
+    private fun Breadcrumb.isValidForRRWebSpan(): Boolean {
+        return !(data["url"] as? String).isNullOrEmpty() &&
+            "start_timestamp" in data &&
+            "end_timestamp" in data
+    }
+
+    private fun String.snakeToCamelCase(): String {
+        val pattern = "_[a-z]".toRegex()
```

Comment:
IIRC compiling a regex pattern is expensive, maybe worth moving into a static object const?